### ISSUE TITLE
Add additional predicates for TypeSubTypeValue

### ISSUE THREAD

**Issue by j8934893** [gchq/Gaffer#2309]
Addition of predicates to enable validation and filtering against TSV elements.

**Comment by sirNikolai** [comment]
Trying to understand this a little more. The way I understood TSV is that it is more of a descriptor node so if we were to implement something like KoryphePredicate on it then that would be an external operation rather than being a component of TSV. Or is there something I'm missing about this?

**Comment by d47853** [comment]
@sirNikolai,

Yeah you're right. TSV is a value object and therefore we'd need to add KoryphePredicates that would filter the object. @j8934893 Are you able to provide any details around the kind of predicates you're looking for?

**Comment by n3101** [comment]
@j8934893 Hiya, please can I remind you about the question above? We have various predicate related stuff in the backlog, so add anything you have in mind here. Otherwise I will close this issue for now.
